js/packages/phoenix-otel/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@arizeai/phoenix-otel",
+  "version": "0.0.1",
+  "description": "Otel registration and convenience methods",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf dist",
+    "prebuild": "pnpm run clean",
+    "build": "tsc --build tsconfig.json tsconfig.esm.json && tsc-alias -p tsconfig.esm.json",
```

Comment:
Appreciate the ESM only. This will let us do cool things like auto register instrumentation lazily based on installed packages